dashboard/data_access.py

```diff
@@ -0,0 +1,206 @@
+import pandas as pd
+import psycopg2
+from psycopg2._psycopg import connection, cursor
+from dotenv import load_dotenv
+from os import environ as ENV
+import psycopg2.extras
+
+load_dotenv()
+
+conn = psycopg2.connect(
+    database=ENV["DB_NAME"],
+    user=ENV["DB_USER"],
+    password=ENV["DB_PASS"],
+    host=ENV["DB_HOST"],
+    port=ENV["DB_PORT"],
+)
+
+
+cur = conn.cursor()
+
```

Comment:
I think you've forgotten to add this to `if __name__ == "__main__":`

Reply:
Done.